components/SerialDevice.vue

```diff
@@ -523,11 +523,19 @@ const connectToDevice = async () => {
                             commandsQueue.processMspResponse(result!.commandName, result!.data);
                         }
                     });
-                    await Msp.getInstance().sendWithPromise(MSP_COMMANDS.MSP_MOTOR_CONFIG).then((result) => {
-                        if (result) {
-                            commandsQueue.processMspResponse(result!.commandName, result!.data);
-                        }
-                    });
+                    if (serialStore.mspData.type == 'inav') {
+                      await Msp.getInstance().sendWithPromise(MSP_COMMANDS.MSP_MOTOR).then((result) => {
```

Comment:
i would like a mapping here like `getTypeMotorCommand(type: string): MSP_COMMANDS` that way we dont have duplicate code.

Reply:
Will do.